opentreemap/treemap/templates/treemap/partials/filter_dropdown.html

```diff
@@ -0,0 +1,14 @@
+<div class="btn-group">
+    <button type="button" class="btn btn-sm btn-info dropdown-toggle" data-toggle="dropdown">
+        {{ filter.selected_option.label }} <span class="caret"></span>
+    </button>
+    <ul {{ filter.container_attr }} class="dropdown-menu" role="menu">
```

Comment:
Can you add a class of `dropdown-menu-right` to this UL?  It was on _one_ of the filters that got removed in favor of this partial, and fixes issues with the dropdown menu getting cut off by the edge of the screen.